templates/cmd/controller/main.go.tpl

```diff
@@ -87,13 +91,14 @@ func main() {
 	}
 
 	mgr, err := ctrlrt.NewManager(ctrlrt.GetConfigOrDie(), ctrlrt.Options{
-		Scheme:			    scheme,
-		Port:			    port,
-		Host:			    host,
-		MetricsBindAddress: ackCfg.MetricsAddr,
-		LeaderElection:		ackCfg.EnableLeaderElection,
-		LeaderElectionID:   awsServiceAPIGroup,
-		Namespace:		    ackCfg.WatchNamespace,
+		Scheme:                  scheme,
+		Port:                    port,
+		Host:                    host,
+		MetricsBindAddress:      ackCfg.MetricsAddr,
+		LeaderElection:          ackCfg.EnableLeaderElection,
+		LeaderElectionID:        awsServiceAPIGroup,
+		Namespace:               ackCfg.WatchNamespace,
+		LeaderElectionNamespace: ackSystemNamespace,
```

Comment:
I was thinking about adding a `--leader-election-namespace` flag to allow users to set their own.. thoughts?

Reply:
I'm not the sure what the use case for this flag is, or what it actually does under the covers. `operator-sdk` only scaffolds with the below options:

```
        Scheme:                 scheme,
        MetricsBindAddress:     metricsAddr,
        Port:                   9443,
        HealthProbeBindAddress: probeAddr,
        LeaderElection:         enableLeaderElection,
        LeaderElectionID:       leaderElectionID,
```

Reply:
`LeaderElectionNamespace` is the namespace where the `Lease` object is created

Reply:
Without namespace, i'm running into `unable to create controller manager	{"aws.service": "lambda", "error": "unable to find leader election namespace: not running in-cluster, please specify LeaderElectionNamespace"}` - does operator-sdk use leases as well? or maybe configmaps (that's what controller-runtime used before leases introduction)?